public/static/docs/api-reference/index.md

```diff
@@ -0,0 +1,30 @@
+# Python API
+
+When you [install](/doc/install) DVC with an package manager like `pip` or
+`conda`, the `dvc` package becomes available to the corresponding Python
+environment. While most of the package implements our
+[command-line tool](/doc/command-reference), we wrote the `dvc.api` module to
+expose special functions you can use in your Python source code.
+
+> For API use, we **strongly** recommend having `dvc` in a requirements file for
+> your Python project, and installing it with a package manager (in a
+> [virtual environment](https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments)
+> preferably).
+
+To use the API, import the module first with:
```

Comment:
either remove, or state it in a different way - API module is `dvc.api` ... it can be imported in a variety of ways

Reply:
Addressed in https://github.com/iterative/dvc.org/pull/908#pullrequestreview-348449124 above.